docker/linux/amazonlinux.Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM amazonlinux:2
+
+RUN amazon-linux-extras enable python3.8
+
+RUN yum update -y
+RUN yum install -y git make python38 gcc10.x86_64 gcc10-c++.x86_64 binutils
```

Comment:
Is architecture suffix really needed?

Reply:
Actually it isn't, I copy pasted the output of `yum search`

0f5677723e6274c90acbcf7fd55d81f6d7aeb179